docs/reference/cluster/state.asciidoc

```diff
@@ -68,17 +75,12 @@ GET /_cluster/state/{metrics}/{indices}
     Shows the `routing_table` part of the response. If you supply a comma
     separated list of indices, the returned output will only contain the routing
     table for these indices.
+    
+`version`::
+    Shows the cluster state version.
 
-`metadata`::
-    Shows the `metadata` part of the response. If you supply a comma separated
-    list of indices, the returned output will only contain metadata for these
-    indices.
-
-`blocks`::
-    Shows the `blocks` part of the response.
-
-`_all`::
-    Shows all metrics.
+[[cluster-state-api-example]]
+==== {api-examples-title}
```

Comment:
It would be great to include a response example in this section. Fine if not though.

Reply:
I'll fire up a three-node cluster and add the example response later. I made a note on it.

Reply:
Sounds good. Feel free to leave it out for now.